pkg/api/routes.go

```diff
@@ -60,6 +60,12 @@ func (h *Handler) GetRoutes() []Route {
 			h.CreateImage,
 		},
 		{
+			"DeleteImage",
+			"POST",
```

Comment:
It is just odd that we are not using the REST verbs appropriate when we are trying to do a restful API. Is the block image information that you are passing an ID? Can that not be passed by doing `DELETE /image/{image_id}`?